docs/data-model.md

```diff
@@ -13,7 +13,9 @@ Installations are only specific to the Jira app, 1 per Jira instance.  It holds
 
 ### GitHubServerApp
 
-Each GitHub App created in a GitHub Enterprise Server instance is added to this table to keep track of them.  We do not hold the GitHub Cloud app in this table since we need it as a default, but still use the same data structure in the code.  Each App will have access to many Organizations under it and we much show all results to the users when needed, which means multiple API calls .
+Each GitHub App created in a GitHub Enterprise Server instance is added to this table to keep track of them.  A Github app _can only be associated with one Jira instance for security reasons_.  This means that each new Jira instance needs to create a new GHE app.
```

Comment:
"for security reasons" is correct, of course, but it's so often used as a blanket argument that I would want a sentence here that explains why exactly (because each Jira instance can be owned by a different customer and we don't want to allow accidental communication from a GitHub org of customer A with a Jira instance of customer B)

Reply:
yeah, good point.